LayoutTests/fast/canvas/canvas-imagedata-dimension-no-overflow.html

```diff
@@ -0,0 +1,45 @@
+<head>
+<script src="../../resources/js-test-pre.js"></script>
+</head>
+<body>
+<script>
+var canvas = document.createElement('canvas');
+var ctx = canvas.getContext('2d');
+canvas.width = 8192;
+canvas.height = 8190;
+ctx.fillStyle ='rgba(0, 255, 0, 1)';
+ctx.fillRect(0, 0, canvas.width, canvas.height);
+var result = ctx.getImageData(0, 0, 1, 1);
+console.log(result);
+var data = result.data;
+console.log('Drawing canvas with MaxSize and getting ImageData from image');
+// expect that d[0] == 0, d[1] == 255, d[2] ==  0, d[3] == 255;
```

Comment:
This comment was meant for you to understand what you need to do.
Now that you have added the assert, you can remove it, it doesn't provide any clarification.

Reply:
Fixed